options.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const tsServerOptionName = 'ts-server';
+const tsProjectOptionName = 'ts-project';
+const sapUserOptionName = 'sap-username';
+const extendedUriFilterOptionName = 'extended-uri-filter';
+const allOptions = [tsServerOptionName, tsProjectOptionName, sapUserOptionName, extendedUriFilterOptionName];
+
+function loadDataAndListen() {
+    chrome.storage.local.get(allOptions, result => {
+        allOptions.forEach(optionName => {
```

Comment:
If the renaming above makes sense, then:
```suggestion
        allOptionIds.forEach(optionId => {
```

Reply:
done